QModManager/API/SMLHelper/Handlers/CraftDataHandler.cs

```diff
@@ -0,0 +1,413 @@
+﻿namespace QModManager.API.SMLHelper.Handlers
+{
+    using Crafting;
+    using Patchers;
+    using Interfaces;
+
+    /// <summary>
+    /// A handler class for adding and editing crafted items.
+    /// </summary>
+    public class CraftDataHandler : ICraftDataHandler
```

Comment:
Suggestion: `TechTypeHandler.AddTechType` could return an instance of a `CustomTechType` class, and this class could have all `CraftDataHandler` calls (plus some others) as methods which are called directly on the object.

Reply:
I can see where you're coming from. As always, this will be something added for convenience but won't fully replace the original methods.  

Since people are really embracing the Asset classes for quickly handling the boilerplate for `Craftable` and `Buildable` items, will this new class integrate nicely with those or will it have to be fully separate?   
Let's take a moment to see if rather than create a new class, we can either add these methods onto  the `Spawnable` class itself or via some extension methods so we don't need to add yet another thing to keep track of.

Reply:
These classes already have properties defining these values. I don't think additional methods would be required